extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -205,12 +206,17 @@ export class SafeframeHostApi {
           'readCookie': false,
           'writeCookie': false,
         }));
+    const ampDoc = Services.documentInfoForDoc(
+        this.baseInstance_.getAmpDoc());
+    const restricted = !!this.win_.document.querySelector(
+        "meta[name='referrer'][content='origin']");
```

Comment:
Why are we only restricting on `origin` and not, say, `no-referrer`?  Can you add a comment describing your reasoning here?

(And maybe link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta and say we're trying to replicate that behavior, if that's what we're doing?)

Reply:
Done